modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/execution/MethodArgumentResolverDelegate.java

```diff
@@ -246,6 +249,8 @@ private void initArgumentResolvers() {
       Object parameterValue = valueSupplier.get();
       if (parameterValue == null) {
         return resolvePrimitiveTypeDefaultValue(parameterType);
+      } else if (parameterType.equals(InputStream.class) && parameterValue instanceof CursorStream) {
```

Comment:
Makes any sense to ask if the parameter type is a InputStream?

Reply:
solved